base/tests/unittests/com/google/idea/blaze/base/run/smrunner/BlazeXmlSchemaTest.java

```diff
@@ -174,6 +174,29 @@ public void testErrorWithoutErrorContent() {
     assertThat(BlazeXmlSchema.getErrorContent(testCase.errors.get(0))).isNull();
   }
 
+  @Test
+  public void testErrorWithoutMessage() {
+    TestSuite parsed =
+        parseXml(
+            "<?xml version='1.0' encoding='UTF-8'?>",
+            "<testsuites>",
+            "  <testsuite name='com.google.ConfigTest' tests='1' failures='0' errors='1'>",
+            "    <testcase name='testCase1' status='run' duration='55' time='55'>",
+            " <error type='java.lang.NullPointerException'><![CDATA[java.lang.NullPointerException",
+            "  at com.example.MyClass.myMethod(MyClass.java:55)",
+            "]]></error>",
+            "    </testcase>",
+            "  </testsuite>",
+            "</testsuites>");
+
+    ErrorOrFailureOrSkipped error = parsed.testSuites.get(0).testCases.get(0).errors.get(0);
+
+    assertThat(error.type).isEqualTo("java.lang.NullPointerException");
+    assertThat(BlazeXmlSchema.getErrorContent(error))
+        .isEqualTo("java.lang.NullPointerException\n  at com.example.MyClass.myMethod(MyClass.java:55)");
+    assertThat(error.message).isNull();
+  }
```

Comment:
This test doesn't actually cover my code change above, and I didn't find any existing tests with coverage of the `BlazeXmlToTestEventsConverter` file.

However, it looks there's prior art for simply adding tests here. At the very least, I think it makes sense to document this scenario as test coverage: https://github.com/bazelbuild/intellij/commit/47b0fa9f0614f131f220fcdabf9276edd0c0be5e